Makefile

```diff
@@ -138,4 +142,8 @@ test-e2e-single-node: install-go-junit-report
 	set -o pipefail; go test -tags=$(GOTAGS) -failfast -timeout 120m -v$${WHAT:+ -run="$$WHAT"} ./test/e2e-single-node/ | ./hack/test-with-junit.sh $(@)
 
 bootstrap-e2e: install-go-junit-report install-setup-envtest
-	set -o pipefail; go test -tags=$(GOTAGS) -v$${WHAT:+ -run="$$WHAT"} ./test/e2e-bootstrap/ | ./hack/test-with-junit.sh $(@)
+	@echo "Setting up KUBEBUILDER_ASSETS"
```

Comment:
thought (non-blocking): I think this setup step may be better under the `install-setup-envtest` target, but if it's too difficult to make that work, then don't worry about it.

Reply:
thats what i was trying to do initially, but didnt get it to work. ill try a new approach but if i cant get it, i wont sweat it

Reply:
resolving this- zack and i decided to stick with keeping it under the bootstrap-e2e target